src/content/en/photon/start.md

```diff
@@ -136,7 +136,7 @@ The SETUP button serves two functions after the Photon has booted:
 
 The SETUP button serves four functions before the Photon has booted:
 
-- Hold down the SETUP button, tap on the RESET button and wait for *less than 3 seconds* to enter *User Safe Mode* mode, where the RGB LED will be flashing Magenta. Release the SETUP button. This is the most frequently mode, because your user application will not execute.  This can be useful if you have a bug in your code that is keeping you from re-flashing it over the air, or causing other hanging issues.
+- Hold down the SETUP button, tap on the RESET button and wait for *less than 3 seconds* to enter *User Safe Mode* mode, where the RGB LED will be flashing Magenta. Release the SETUP button. This mode places the device in a safe state where no user code is run. This can be useful if you have a bug in your code that is keeping you from re-flashing it over the air, or causing other hanging issues.
```

Comment:
This text originally said: "In this mode your user application will not execute."  BLAME: https://github.com/rephorm/docs/commit/6b97043bd0ca346e3ac4ea02c704d4711816913a